sound/soc/sof/topology.c

```diff
@@ -988,25 +988,28 @@ static int sof_parse_word_tokens(struct snd_soc_component *scomp,
  * @object: target ipc struct for parsed values
  * @tokens: token definition array describing what tokens to parse
  * @count: number of tokens in definition array
- * @array: source pointer to consecutive vendor arrays to be parsed
- * @priv_size: total size of the consecutive source arrays
+ * @array: source pointer to consecutive vendor arrays in topology
+ * @array_size: total size of the vendor arrays
  * @sets: number of similar token sets to be parsed, 1 set has count elements
  * @object_size: offset to next target ipc struct with multiple sets
+ * @tuples: array of tuples to add the parsed tuples to
+ * @num_tuples: pointer to the number of tuples in the tuples array
  *
  * This function parses multiple sets of tokens in vendor arrays into
  * consecutive ipc structs.
  */
 static int sof_parse_token_sets(struct snd_soc_component *scomp,
 				    void *object, const struct sof_topology_token *tokens,
 				    int count, struct snd_soc_tplg_vendor_array *array,
-				    int priv_size, int sets, size_t object_size)
+				    int array_size, int sets, size_t object_size,
+				    struct snd_sof_tuple *tuples, int *num_tuples)
 {
 	size_t offset = 0;
 	int found = 0;
 	int total = 0;
 	int asize;
 
-	while (priv_size > 0 && total < count * sets) {
+	while (array_size > 0 && total < count * sets) {
```

Comment:
same here, if you are going a renaming, this should land in a separate patch. That way you only have the signature change in the second patch, much easier to review.